swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    
+    def __init__(self, inventory=[]):
+        self.inventory = inventory 
+
+    def add(self, item):
+        #self.item = item
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        #self.item = item
+        if item in self.inventory:
+            self.inventory.remove(item)
+            return item 
+        else: 
+            return False
+
+    def get_by_category(self, category):
+        list_of_categories = []
+        for item in self.inventory:
+            if item.category == category:
+                list_of_categories.append(item)
+        return list_of_categories
+
+    def swap_items(self, other_vendor, my_item, their_item): 
+        if my_item not in self.inventory or their_item not in other_vendor.inventory:
+            return False
+        else:
+            self.inventory.remove(my_item)
+            other_vendor.inventory.append(my_item)
+            other_vendor.remove(their_item)
+            self.inventory.append(their_item)
```

Comment:
this works! but why not use the instance methods `add` and `remove` we've already made?

```suggestion
            self.remove(my_item)
            other_vendor.add(my_item)
            other_vendor.remove(their_item)
            self.add(their_item)
```

we can also merge these lines together. if we look, `remove` returns the item it removes, so why not pass the entire remove function inside add? `self.add(other_vendor.remove(their_item))`